Tools/Schema.NET.Tool/GeneratorModels/GeneratorSchemaEnumeration.cs

```diff
@@ -8,6 +8,11 @@ namespace Schema.NET.Tool.GeneratorModels
     public class GeneratorSchemaEnumeration : GeneratorSchemaObject
 #pragma warning restore CA1724 // Identifiers should conflict with namespaces
     {
+        public GeneratorSchemaEnumeration(string name, string layer)
```

Comment:
I started to do this too and then reverted. In fact, I tried using `record` instead but got an error because the project is `netstandard2.0`.